TestSuites/ProjectSuit.robot

```diff
@@ -2,16 +2,15 @@
 Library           Selenium2Library
 Resource          ../service/UI/ProjectService.robot
 Resource          ../globalConfig/Common.robot
-Library           ../utils/TestDataGen.py
-Library           ../page/ProjectPage.py
 Resource          ../service/UI/LoginService.robot
+Resource          ../utils/TestDataGenerator.robot
 
 *** Test Cases ***
 CreateProject
     [Documentation]    The test is intended to check the creation of a project via Project Page.
     [Setup]    Run Keywords    Set Random String
     ...    AND    Login And Check Top Logo Visible UI    ${ADMIN_LOGIN}    ${ADMIN_PASSWORD}
```

Comment:
Replace with Login UI or Login And Wait Untill Completed. Remove 'check' option from this setup